nsxt/resource_nsxt_project_ip_address_allocation_test.go

```diff
@@ -0,0 +1,190 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestProjectIpAddressAllocationCreateAttributes = map[string]string{
+	"display_name":    getAccTestResourceName(),
+	"description":     "terraform created",
+	"allocation_size": "1",
+}
+
+var accTestProjectIpAddressAllocationUpdateAttributes = map[string]string{
+	"display_name":    getAccTestResourceName(),
+	"description":     "terraform updated",
+	"allocation_size": "1",
+}
+
+func TestAccResourceNsxtProjectIpAddressAllocation_basic(t *testing.T) {
+	testResourceName := "nsxt_policy_project_ip_address_allocation.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t); testAccOnlyMultitenancy(t) },
```

Comment:
Better to run these with the `testAccOnlyVPC` precheck as for now we don't have a multitenancy job for vpc20 branch.